kernel/binaries/betatest.txt

```diff
@@ -0,0 +1,5 @@
+PW: betatest
```

Comment:
please remove this file and winpmem_old.ico below

Reply:
Could we agree not to change the prototype function already in this pull request, and have a separate one?